alfajores.env

```diff
@@ -2,7 +2,7 @@
 #                                ↓ REQUIRED ↓                                 #
 ###############################################################################
 
-# Network to run the node on ("celo-mainnet","alfajores" or "baklava")
```

Comment:
Using "mainnet" will make it harder to merge our repo with the upstream one. Which probably won't happen anyway, so I guess we can do it.